front/src/js/message-list.js

```diff
@@ -3,15 +3,27 @@ const messages = {
     props: ['messages'],
     methods: {
         humanDate(ts,now){
+            console.log(ts);
             return moment(ts).fromNow();
+        },
+        async getAuthor(id){
+            return await this.srv.getUserByid(id);
         }
+
     },
     data: function(){
         return {
-            now: new Date()
+            now: new Date(),
+            srv: new ScratchyService("http://localhost:5000/api")
```

Comment:
Remove it's unused now that the data provided by the parent component already contains author details